src/helpers/ComponentRegister.test.tsx

```diff
@@ -2,21 +2,34 @@ import React from "react";
 
 import {
   ComponentRegister,
-  LinkComponent,
-  LinkComponentProps,
   defaultComponentRegisterOptions
 } from "./ComponentRegister";
+import { LinkComponentType, LinkComponentTypeProps } from "./LinkComponentType";
+import {
+  LinkButtonComponentType,
+  LinkButtonComponentTypeProps
+} from "./LinkButtonComponentType";
+
+const TestLinkComponent: LinkComponentType = ({
+  href,
+  children
+}: LinkComponentTypeProps) => <div data-href={href}>{children}</div>;
 
-const TestLinkComponent: LinkComponent = ({
+const TestLinkButtonComponent: LinkButtonComponentType = ({
   href,
   children
-}: LinkComponentProps) => <div data-href={href}>{children}</div>;
+}: LinkButtonComponentTypeProps) => (
+  <a role="button" href={href}>
```

Comment:
This is very similar to the real implementation. Can we use something that isn't so we can be confident about the tests. Suggest another `div`.